docs/guide/localization/README.md

```diff
@@ -0,0 +1,135 @@
+
+<AposCodeBlock>
+  ```javascript
+    module.exports = {
+      options: {
+        locales: {
+          en: {
+            label: 'English'
+          },
+          es: {
+            label: 'Spanish'
+          },
+          'es-MX': {
+            label: 'Spanish (Mexico)'
+          },
+          'fr-CA': {
+            label: 'French (Canada)'
+          },
+          'en-CA': {
+            label: 'English (Canada)'
+          }
+        }
+      }
+    }
+  ```
+  <template v-slot:caption>
+    modules/@apostrophecms/i18n/index.js
+  </template>
+</AposCodeBlock>
+
+::: note
+- Locale names (e.g., `'en'`, `'fr-CA'`) must begin with an alphabetic (non-numeric) character.
+- The best practice for locale names is to use a two-character language code (`'en'`) or the language code with two character country code, capitalized (`'en-GB'`). This will improve compatibility with i18n features as they are added to Apostrophe.
+:::
+
+There's one thing left to configure these locales for use: We need to tell Apostrophe how to identify which one to use. This is done based on the URL used to access the website, either by the URL **hostname**, the URL path **prefix**, or a combination of the two.
+
+<AposCodeBlock>
+  ```javascript
+    module.exports = {
+      options: {
+        locales: {
+          en: {
+            label: 'English'
+          },
+          es: {
+            label: 'Spanish',
+            prefix: '/es'
+          },
+          'es-MX': {
+            label: 'Spanish (Mexico)',
+            hostname: 'example.mx'
+          },
+          'fr-CA': {
+            label: 'French (Canada)'
+            hostname: 'example.ca',
+            prefix: '/fr'
+          },
+          'en-CA': {
+            label: 'English (Canada)'
+            hostname: 'example.ca',
+            prefix: '/en'
+          }
+        }
+      }
+    }
+  ```
+  <template v-slot:caption>
+    modules/@apostrophecms/i18n/index.js
+  </template>
+</AposCodeBlock>
+
+::: note
+If `hostname` is used for any locale:
+- a `baseUrl` must be set on the application, defining the default hostname, OR
+- a `hostname` setting must be used on all locales
+
+For this example we'll assume we have `baseUrl: 'example.com'` set on the application.
+:::
+
+So how does Apostrophe choose the best locale to use? In many cases it is clear. If there is conflict, however, the best locale uses the following prioritization:
+1. The locale matches *both* the URL's `hostname` *and* path `prefix`.
+2. The locale matches the URL's `hostname` only.
+3. The locale matches the URL's path `prefix` only.
```

Comment:
"The URL matches the locale's configured `prefix` and the locale has no `hostname`."